source/03-components/call-to-action/_call-to-action.scss

```diff
@@ -0,0 +1,24 @@
+// @file
+// Component: c-call-to-action
+
+@use '00-config' as *;
+
+.c-call-to-action {
+  .l-grid {
+    align-items: center;
+  }
```

Comment:
@jbwagner Not a huge fan of adding non-call-to-action classes in here. Is there another way to achieve this? Maybe with an extra wrapper around the grid?

Reply:
Added a modifier class to the grid component, should be a bit more self-contained [a8cd193](https://github.com/forumone/gesso/pull/886/commits/a8cd193de80ee2f4feea7c09dc6c934c7bb7fffe)